### ISSUE TITLE
Interface Conversion Panic

### ISSUE THREAD

**Issue by zanes2016** [mesos/mesos_exporter#90]
We're running the latest docker version ([mesosphere/mesos_exporter:latest](https://hub.docker.com/r/mesosphere/mesos_exporter/)) and we're seeing the following panic:

```
panic: interface conversion: prometheus.Collector is *prometheus.GaugeVec, not *main.settableCounterVec

goroutine 15 [running]:
main.newMasterCollector.func20(0xc0001526f0, 0x8f4be0, 0xc00000e0e0, 0x0, 0x0)
	/go/src/github.com/mesosphere/mesos_exporter/master.go:419 +0x6f1
main.(*metricCollector).Collect(0xc000073c00, 0xc0000888a0)
	/go/src/github.com/mesosphere/mesos_exporter/common.go:295 +0x1e7
github.com/mesosphere/mesos_exporter/vendor/github.com/prometheus/client_golang/prometheus.(*Registry).Gather.func2(0xc000020d60, 0xc0000888a0, 0x8f4de0, 0xc000073c00)
	/go/src/github.com/mesosphere/mesos_exporter/vendor/github.com/prometheus/client_golang/prometheus/registry.go:383 +0x61
created by github.com/mesosphere/mesos_exporter/vendor/github.com/prometheus/client_golang/prometheus.(*Registry).Gather
	/go/src/github.com/mesosphere/mesos_exporter/vendor/github.com/prometheus/client_golang/prometheus/registry.go:381 +0x302
```

It seems to be due to whatever change was pushed into the build about 2 days ago.

**Comment by jacobjanco** [comment]
https://github.com/mesos/mesos_exporter/issues/87

**Comment by jacobjanco** [comment]
https://github.com/mesos/mesos_exporter/pull/91

**Comment by jpeach** [comment]
Fixed by #86
